src/components/Comments/comment.css

```diff
@@ -0,0 +1,103 @@
+@import url("https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600,700,800");
+/* ----- Coments ----- */
+.comments__title {
+  margin-top: 40px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-align: center;
+  color: #2196f3;
+  font-family: "Fira Sans", sans-serif;
+}
+
+/* ----- Comment ----- */
+.list-comment__comment {
+  text-align: center;
+  list-style: none;
+  margin-bottom: 150px; 
+  display: table;
+  content: "";
+  font-family: "Fira Sans", sans-serif;
+
+}
+
+/* ----- Comment text ----- */
+.comment__comment-text {
+  width: 400px;
+  left: 50%;
+  margin-left: -200px;
+  position: absolute;
+  border-radius: 25px; 
+  background-color: #2196f3;
+  color: white;
+  font-family: "Fira Sans", sans-serif;
+}
+
+.comment-text__name-author {
+  margin: 15px 15px;
+   font-weight: 500; 
+   color: white;
+   font-family: "Fira Sans", sans-serif;
+}
+
+.name-author__data-post {
+  font-weight: 400;
+  font-size: 10pt;
+  color: white;
+  font-family: "Fira Sans", sans-serif;
+}
+
+.name-author__data-post::before {
+  content: " · ";
```

Comment:
Why are we applying this content? Could do with a comment to explain what this is doing